src/Application/Auth/Policies/BusinessPolicy.php

```diff
@@ -66,15 +66,8 @@ public function index(User $user)
      */
     public function update(User $user, Business $business)
     {
-        if ($this->userIsGuest($user)) {
-            return false;
-        }
-
-        if ($business->isPublished() && $this->userIsRestarter($user)) {
```

Comment:
This should remain (but checking if a user is an Editor now).

If the business is published, then Editors can't update it.

Reply:
Ok.  Added something for this.